securedrop/journalist_templates/col.html

```diff
@@ -4,7 +4,7 @@
 
   <p class="breadcrumbs">
     <a href="/">{{ gettext('All Sources') }}</a>
-    <i class="fa fa-chevron-right"></i>
+    <img src="{{ url_for('static', filename='icons/chevron-right.png') }}" class="icon" width="9" height="14" alt=">">
```

Comment:
And here, I think I would use `alt=""` on the separator icon rather than have a screenreader say something like "link all sources greater than contemptible bodybuilding" (or whatever the source designation were, and as I don't have a screenreader set up at the moment, this is just a guess at what it might sound like).

Really, the breadcrumbs should be done in a more [semantic and accessible way](https://www.w3.org/TR/wai-aria-practices/#breadcrumb), but that's definitely out of scope for this PR. :slightly_smiling_face:

Reply:
`alt=""` used.